guides/common/modules/proc_using-red-hat-insights.adoc

```diff
@@ -24,6 +24,16 @@ On each recommendation line, the vertical ellipsis provide a *View in {Team} Ins
 If the host has recommendations listed, click on the number of recommendations.
 On the *Insights* tab, the vertical ellipsis next to the *Remediate* button provide a *Go To {Project} Insights page* link to information for the system, and a *View in {Team} Insights* link to host details on the console.
 
+.Excluding hosts from rh-cloud and insights-client reports
+Starting from {Project} {ProductVersion}, you can omit host reporting from _rh-cloud_ and _insight-client_ reports by setting the `host_registration_insights` parameter to *False*.
+{Project} will exclude the hosts from _rh-cloud_ reports and block _insight-client_ from uploading a report to the cloud.
+
+This parameter can also be set at the organization, hostgroup, subnet, and domain level.
+It automatically prevents new hosts from being uploaded as long as they are associated with the entity.
+
+In the case where the host is already reported while setting the parameter to false, it can still be removed by the aging process from the cloud.
```

Comment:
```suggestion
If a host already sends reports to {Project} and you set the parameter to false, it can still be removed by the aging process from the cloud.
```

I don't understand the last part of the sentence. Which cloud? Where? What's the "aging process"?

Reply:
Addressed.